src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -320,10 +331,45 @@ private void FinishBindingOnUIThread(IProgressController controller, IProgressSt
             }
         }
 
+        internal /*for testing purposes*/ void UnpackAdditionalFiles(IProgressController controller, CancellationToken token, IProgressStepExecutionEvents notificationEvents)
+        {
+            if (!this.AdditionalFiles.Any())
+            {
+                return; // Nothing to unpack
+            }
+
+            Solution2 solution = this.projectSystemHelper.GetCurrentActiveSolution();
+            if (solution == null)
```

Comment:
An assert will be enough IMO. Otherwise you have a bug and a missing test case (need to return a from this if block, otherwise you will get a null ref exception).